kiva/agg/tests/test_save.py

```diff
@@ -3,14 +3,12 @@
 
 from numpy import allclose, ravel
 
-import nose
-
 from kiva import agg
 
 
 # FIXME:
 #   These tests are broken, and Peter promised to fix it at some point.
```

Comment:
Might be better to just delete these tests? Peter Wang hasn't worked on this code for a long time. I'm _nearly certain_ that he has no interest in fixing these tests.

Reply:
I noticed that only two of the four tests actually fail. It could also be a platform dependent behaviour, though.

Reply:
FTR, these are the two tests that fail:
```
======================================================================
FAIL: test_bgr24_format (kiva.agg.tests.test_save.Test_Save)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/kchoi/.edm/envs/enable-test-3.6-pyqt5-pillow/lib/python3.6/site-packages/kiva/agg/tests/test_save.py", line 23, in test_bgr24_format
    self.do_check_format('bgr24')
  File "/Users/kchoi/.edm/envs/enable-test-3.6-pyqt5-pillow/lib/python3.6/site-packages/kiva/agg/tests/test_save.py", line 43, in do_check_format
    self.format_output_map[fmt])
AssertionError: Lists differ: [255, 255, 255, 255, 255, 255, 255, 0, 0, 255, 0, 0] != [255, 255, 255, 255, 255, 255, 0, 0, 255, 0, 0, 255]

First differing element 6:
255
0

- [255, 255, 255, 255, 255, 255, 255, 0, 0, 255, 0, 0]
?                                              ------

+ [255, 255, 255, 255, 255, 255, 0, 0, 255, 0, 0, 255]
?                                ++++++


======================================================================
FAIL: test_bgra32_format (kiva.agg.tests.test_save.Test_Save)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/kchoi/.edm/envs/enable-test-3.6-pyqt5-pillow/lib/python3.6/site-packages/kiva/agg/tests/test_save.py", line 29, in test_bgra32_format
    self.do_check_format('bgra32')
  File "/Users/kchoi/.edm/envs/enable-test-3.6-pyqt5-pillow/lib/python3.6/site-packages/kiva/agg/tests/test_save.py", line 43, in do_check_format
    self.format_output_map[fmt])
AssertionError: Lists differ: [255, 255, 255, 255, 255, 255, 255, 255, 255, 0, 0, 255, 255, 0, 0, 255] != [255, 255, 255, 255, 255, 255, 255, 255, 0, 0, 255, 255, 0, 0, 255, 255]

First differing element 8:
255
0

- [255, 255, 255, 255, 255, 255, 255, 255, 255, 0, 0, 255, 255, 0, 0, 255]
?                                               ------          ------

+ [255, 255, 255, 255, 255, 255, 255, 255, 0, 0, 255, 255, 0, 0, 255, 255]
?                                          ++++++          ++++++
```